packages/no-payment-integration/project.json

```diff
@@ -0,0 +1,22 @@
+{
+    "root": "packages/no-payment-integration",
+    "sourceRoot": "packages/no-payment-integration/src",
+    "projectType": "library",
+    "targets": {
+        "lint": {
+            "executor": "@nrwl/linter:eslint",
+            "outputs": ["{options.outputFile}"],
+            "options": {
+                "lintFilePatterns": ["packages/no-payment-integration/**/*.ts"]
+            }
+        },
+        "test": {
+            "executor": "@nrwl/jest:jest",
+            "outputs": ["coverage/packages/no-payment-integration"],
+            "options": {
+                "jestConfig": "packages/no-payment-integration/jest.config.js"
+            }
+        }
+    },
+    "tags": ["scope:shared"]
```

Comment:
Setting the scope to `shared` because 'core' still imports this method for testing purposes.